weather_sp/splitter_pipeline/file_name_utils_test.py

```diff
@@ -14,58 +14,86 @@
 
 import unittest
 
-from .file_name_utils import get_output_file_base_name
+from .file_name_utils import get_output_file_info, OutFileInfo
 
 
 class FileNameUtilsTest(unittest.TestCase):
 
-    def test_get_output_file_base_name_format(self):
-        out_info = get_output_file_base_name(filename='gs://my_bucket/data_to_split/2020/01/21.nc',
-                                             out_pattern='gs://my_bucket/splits/{2}-{1}-{0}_old_data.',
-                                             out_dir=None,
-                                             input_base_dir='ignored')
-        self.assertEqual(out_info.file_name_template, 'gs://my_bucket/splits/2020-01-21_old_data.')
-        self.assertEqual(out_info.ending, '.nc')
+    def test_get_output_file_info_pattern(self):
+        out_info = get_output_file_info(filename='gs://my_bucket/data_to_split/2020/01/21.nc',
```

Comment:
nit (optional): I see the common convention of `actual` vs `expected` pretty often.

Reply:
Done.